### ISSUE TITLE
Dune dashboard integration

### ISSUE THREAD

**Issue by annamehr** [centrifuge/apps#2016]
## Background
To maintain our position as the leading RWA protocol, Data accessibility is key and that's why Anemoy's LTF data should be available on Dune. 

## Requirements
* Support Dune team accessing data
